pkg/controller/razeedeployment/razeedeployment_controller.go

```diff
@@ -181,24 +156,21 @@ func (r *ReconcileRazeeDeployment) Reconcile(request reconcile.Request) (reconci
 	// Check if the RazeeDeployment instance is being marked for deletion
 	isMarkedForDeletion := instance.GetDeletionTimestamp() != nil
 	if isMarkedForDeletion {
-		if utils.Contains(instance.GetFinalizers(), razeeDeploymentFinalizer) {
-			//Run finalization logic for the razeeDeploymentFinalizer.
+		if utils.Contains(instance.GetFinalizers(), utils.RAZEE_DEPLOYMENT_FINALIZER) {
+			//Run finalization logic for the RAZEE_DEPLOYMENT_FINALIZER.
 			//If it fails, don't remove the finalizer so we can retry during the next reconcile
-			return r.partialUninstall(instance)
+			return r.finalizeRazeeDeployment(instance)
```

Comment:
Regression, revert to partialUninstall.
```suggestion
			return r.partialUninstall(instance)
```